packages/oauth/src/index.ts

```diff
@@ -636,7 +739,11 @@ export type OrgInstallationQuery = InstallationQuery<true>;
 // of Bolt.
 export interface AuthorizeResult {
   botToken?: string;
+  botRefreshToken?: string;
+  botTokenExpiresAt?: number;
```

Comment:
How about having a comment like `// UTC time in seconds` for the expiresAt properties?